### ISSUE TITLE
pip install fail

### ISSUE THREAD

**Issue by Houyikai** [Nixtla/neuralforecast#743]
### What happened + What you expected to happen

i create a conda envirment with python=3.10 and use pip install neuralforecast , then i run the exact code which were give in the readme get started  and i get an error said no matplotlib, after conda install matplotlib, a new error pop out said module 'transformers' has no attribute 'configuration_utils'.  

### Versions / Dependencies

# packages in environment at D:\code\env\neuralforecast:
#
# Name                    Version                   Build  Channel
abseil-cpp                20211102.0           h0e60522_0    conda-forge
aiohttp                   3.8.5                    pypi_0    pypi
aiohttp-cors              0.7.0                      py_0    conda-forge
aiosignal                 1.3.1              pyhd8ed1ab_0    conda-forge
asttokens                 2.2.1              pyhd8ed1ab_0    conda-forge
async-timeout             4.0.3              pyhd8ed1ab_0    conda-forge
attrs                     23.1.0             pyh71513ae_1    conda-forge
backcall                  0.2.0              pyh9f0ad1d_0    conda-forge
backports                 1.0                pyhd8ed1ab_3    conda-forge
backports.functools_lru_cache 1.6.5              pyhd8ed1ab_0    conda-forge
blas                      1.0                         mkl  
bottleneck                1.3.5           py310h9128911_0  
brotli                    1.0.9                h2bbff1b_7  
brotli-bin                1.0.9                h2bbff1b_7  
brotlipy                  0.7.0           py310he2412df_1004    conda-forge
bzip2                     1.0.8                he774522_0  
c-ares                    1.19.0               h2bbff1b_0  
ca-certificates           2023.7.22            h56e8100_0    conda-forge
cachetools                5.3.1              pyhd8ed1ab_0    conda-forge
certifi                   2023.7.22                pypi_0    pypi
cffi                      1.15.1          py310hcbf9ad4_0    conda-forge
charset-normalizer        3.2.0                    pypi_0    pypi
click                     8.1.7                    pypi_0    pypi
colorama                  0.4.6              pyhd8ed1ab_0    conda-forge
colorful                  0.5.4              pyhd8ed1ab_0    conda-forge
comm                      0.1.4              pyhd8ed1ab_0    conda-forge
contourpy                 1.0.5           py310h59b6b97_0  
cryptography              41.0.2          py310hac1b9e3_0  
cycler                    0.11.0             pyhd3eb1b0_0  
dataclasses               0.8                pyhc8e2a94_3    conda-forge
debugpy                   1.6.7           py310hd77b12b_0  
decorator                 5.1.1              pyhd8ed1ab_0    conda-forge
distlib                   0.3.7              pyhd8ed1ab_0    conda-forge
exceptiongroup            1.1.3              pyhd8ed1ab_0    conda-forge
executing                 1.2.0              pyhd8ed1ab_0    conda-forge
filelock                  3.12.3             pyhd8ed1ab_0    conda-forge
fonttools                 4.25.0             pyhd3eb1b0_0  
freetype                  2.12.1               ha860e81_0  
frozenlist                1.4.0                    pypi_0    pypi
fsspec                    2023.9.0           pyh1a96a4e_0    conda-forge
giflib                    5.2.1                h8cc25b3_3  
glib                      2.69.1               h5dc1a3c_2  
google-api-core           2.11.1             pyhd8ed1ab_0    conda-forge
google-auth               2.22.0             pyh1a96a4e_0    conda-forge
googleapis-common-protos  1.60.0             pyhd8ed1ab_0    conda-forge
grpc-cpp                  1.48.2               hfe90ff0_1  
grpcio                    1.57.0                   pypi_0    pypi
icu                       58.2                 ha925a31_3  
idna                      3.4                pyhd8ed1ab_0    conda-forge
importlib-metadata        6.8.0              pyha770c72_0    conda-forge
importlib_metadata        6.8.0                hd8ed1ab_0    conda-forge
importlib_resources       6.0.1              pyhd8ed1ab_0    conda-forge
intel-openmp              2023.1.0         h59b6b97_46319  
ipykernel                 6.25.1             pyh6817e22_0    conda-forge
ipython                   8.15.0             pyh5737063_0    conda-forge
jedi                      0.19.0             pyhd8ed1ab_0    conda-forge
jinja2                    3.1.2              pyhd8ed1ab_1    conda-forge
jpeg                      9e                   h2bbff1b_1  
jsonschema                4.19.0                   pypi_0    pypi
jsonschema-specifications 2023.7.1                 pypi_0    pypi
jupyter_client            8.3.1              pyhd8ed1ab_0    conda-forge
jupyter_core              5.3.1           py310h5588dad_0    conda-forge
kiwisolver                1.4.4           py310hd77b12b_0
krb5                      1.20.1               h5b6d351_0
lerc                      3.0                  hd77b12b_0
libbrotlicommon           1.0.9                h2bbff1b_7
libbrotlidec              1.0.9                h2bbff1b_7
libbrotlienc              1.0.9                h2bbff1b_7
libclang                  14.0.6          default_hb5a9fac_1
libclang13                14.0.6          default_h8e68704_1
libdeflate                1.17                 h2bbff1b_0
libffi                    3.4.4                hd77b12b_0
libiconv                  1.16                 h2bbff1b_2
libpng                    1.6.39               h8cc25b3_0
libpq                     12.15                h906ac69_1
libprotobuf               3.20.3               h23ce68f_0
libsodium                 1.0.18               h8d14728_1    conda-forge
libtiff                   4.5.1                hd77b12b_0
libuv                     1.44.2               h8ffe710_0    conda-forge
libwebp                   1.2.4                hbc33d0d_1
libwebp-base              1.2.4                h2bbff1b_1
libxml2                   2.10.4               h0ad7f3c_1
libxslt                   1.1.37               h2bbff1b_1
lightning-utilities       0.9.0              pyhd8ed1ab_0    conda-forge
lz4-c                     1.9.4                h2bbff1b_0
markupsafe                2.1.3                    pypi_0    pypi
matplotlib                3.7.1           py310haa95532_1
matplotlib-base           3.7.1           py310h4ed8f06_1
matplotlib-inline         0.1.6              pyhd8ed1ab_0    conda-forge
mkl                       2023.1.0         h6b88ed4_46357
mkl-service               2.4.0           py310h2bbff1b_1
mkl_fft                   1.3.6           py310h4ed8f06_1
mkl_random                1.2.2           py310h4ed8f06_1
mpmath                    1.3.0              pyhd8ed1ab_0    conda-forge
msgpack                   1.0.5                    pypi_0    pypi
msgpack-python            1.0.3           py310h59b6b97_0
multidict                 6.0.4                    pypi_0    pypi
munkres                   1.1.4                      py_0
nest-asyncio              1.5.6              pyhd8ed1ab_0    conda-forge
networkx                  3.1                pyhd8ed1ab_0    conda-forge
neuralforecast            1.6.3              pyhd8ed1ab_0    conda-forge
ninja                     1.10.2               haa95532_5
ninja-base                1.10.2               h6d14046_5
numexpr                   2.8.4           py310h2cd9be0_1
numpy                     1.25.2          py310h055cbcc_0
numpy-base                1.25.2          py310h65a83cf_0
opencensus                0.11.2             pyhd8ed1ab_0    conda-forge
opencensus-context        0.1.3           py310h5588dad_1    conda-forge
openssl                   3.0.10               h2bbff1b_2
packaging                 23.1               pyhd8ed1ab_0    conda-forge
pandas                    2.1.0                    pypi_0    pypi
parso                     0.8.3              pyhd8ed1ab_0    conda-forge
pcre                      8.45                 hd77b12b_0
pickleshare               0.7.5                   py_1003    conda-forge
pillow                    9.4.0           py310hd77b12b_0
pip                       23.2.1          py310haa95532_0
pkgutil-resolve-name      1.3.10             pyhd8ed1ab_0    conda-forge
platformdirs              3.10.0             pyhd8ed1ab_0    conda-forge
ply                       3.11            py310haa95532_0
prometheus_client         0.13.1             pyhd8ed1ab_0    conda-forge
prompt-toolkit            3.0.39             pyha770c72_0    conda-forge
prompt_toolkit            3.0.39               hd8ed1ab_0    conda-forge
protobuf                  4.24.2                   pypi_0    pypi
psutil                    5.9.0           py310h2bbff1b_0
pure_eval                 0.2.2              pyhd8ed1ab_0    conda-forge
py-spy                    0.3.14               h90009a2_0
pyarrow                   6.0.1                    pypi_0    pypi
pyasn1                    0.4.8                      py_0    conda-forge
pyasn1-modules            0.2.7                      py_0    conda-forge
pycparser                 2.21               pyhd8ed1ab_0    conda-forge
pydantic                  0.18.2                     py_0    conda-forge
pygments                  2.16.1             pyhd8ed1ab_0    conda-forge
pyopenssl                 23.2.0             pyhd8ed1ab_1    conda-forge
pyparsing                 3.0.9           py310haa95532_0
pyqt                      5.15.7          py310hd77b12b_0
pyqt5-sip                 12.11.0         py310hd77b12b_0
pyrsistent                0.18.0          py310h2bbff1b_0
pysocks                   1.7.1              pyh0701188_6    conda-forge
python                    3.10.12              he1021f5_0
python-dateutil           2.8.2              pyhd8ed1ab_0    conda-forge
python-tzdata             2023.3             pyhd8ed1ab_0    conda-forge
python_abi                3.10                    2_cp310    conda-forge
pytorch                   2.0.1           cpu_py310hb0bdfb8_0
pytorch-lightning         2.0.8              pyhd8ed1ab_0    conda-forge
pytz                      2023.3             pyhd8ed1ab_0    conda-forge
pyu2f                     0.1.5              pyhd8ed1ab_0    conda-forge
pywin32                   305             py310h2bbff1b_0
pyyaml                    6.0.1                    pypi_0    pypi
pyzmq                     23.2.1          py310h73ada01_0    conda-forge
qt-main                   5.15.2               h879a1e9_9
qt-webengine              5.15.9               h5bd16bc_7
qtwebkit                  5.212                h2bbfb41_5
ray                       2.6.3                    pypi_0    pypi
ray-core                  2.3.0           py310h5da7b33_1
ray-default               2.3.0           py310haa95532_1
ray-tune                  2.3.0           py310haa95532_1
re2                       2022.04.01           h0e60522_0    conda-forge
referencing               0.30.2                   pypi_0    pypi
requests                  2.31.0             pyhd8ed1ab_0    conda-forge
rpds-py                   0.10.0                   pypi_0    pypi
rsa                       4.9                pyhd8ed1ab_0    conda-forge
setproctitle              1.2.2           py310he2412df_2    conda-forge
setuptools                68.0.0          py310haa95532_0
sip                       6.6.2           py310hd77b12b_0
six                       1.16.0             pyh6c4a22f_0    conda-forge
smart_open                6.3.0              pyhd8ed1ab_1    conda-forge
sqlite                    3.41.2               h2bbff1b_0
stack_data                0.6.2              pyhd8ed1ab_0    conda-forge
sympy                     1.12               pyh04b8f61_3    conda-forge
tabulate                  0.9.0              pyhd8ed1ab_1    conda-forge
tbb                       2021.8.0             h59b6b97_0
tensorboardx              2.6.2.2            pyhd8ed1ab_0    conda-forge
tk                        8.6.12               h2bbff1b_0
toml                      0.10.2             pyhd3eb1b0_0
torchmetrics              1.1.1              pyhd8ed1ab_0    conda-forge
tornado                   6.2             py310he2412df_0    conda-forge
tqdm                      4.66.1             pyhd8ed1ab_0    conda-forge
traitlets                 5.9.0              pyhd8ed1ab_0    conda-forge
typing-extensions         4.7.1                hd8ed1ab_0    conda-forge
typing_extensions         4.7.1              pyha770c72_0    conda-forge
tzdata                    2023c                h04d1e81_0
urllib3                   2.0.4                    pypi_0    pypi
vc                        14.2                 h21ff451_1
virtualenv                20.24.4            pyhd8ed1ab_0    conda-forge
vs2015_runtime            14.27.29016          h5e58377_2
wcwidth                   0.2.6              pyhd8ed1ab_0    conda-forge
wheel                     0.38.4          py310haa95532_0
win_inet_pton             1.1.0              pyhd8ed1ab_6    conda-forge
xz                        5.4.2                h8cc25b3_0
yaml                      0.2.5                h8ffe710_2    conda-forge
yarl                      1.9.2                    pypi_0    pypi
zeromq                    4.3.4                h0e60522_1    conda-forge
zipp                      3.16.2             pyhd8ed1ab_0    conda-forge
zlib                      1.2.13               h8cc25b3_0
zstd                      1.5.5                hd43e919_0

### Reproduction script

import numpy as np
import pandas as pd
from IPython.display import display, Markdown

import matplotlib.pyplot as plt
from neuralforecast import NeuralForecast
from neuralforecast.models import NBEATS, NHITS
from neuralforecast.utils import AirPassengersDF

# Split data and declare panel dataset
Y_df = AirPassengersDF
Y_train_df = Y_df[Y_df.ds<='1959-12-31'] # 132 train
Y_test_df = Y_df[Y_df.ds>'1959-12-31'] # 12 test

# Fit and predict with NBEATS and NHITS models
horizon = len(Y_test_df)
models = [NBEATS(input_size=2 * horizon, h=horizon, max_steps=50),
          NHITS(input_size=2 * horizon, h=horizon, max_steps=50)]
nf = NeuralForecast(models=models, freq='M')
nf.fit(df=Y_train_df)
Y_hat_df = nf.predict().reset_index()

# Plot predictions
fig, ax = plt.subplots(1, 1, figsize = (20, 7))
Y_hat_df = Y_test_df.merge(Y_hat_df, how='left', on=['unique_id', 'ds'])
plot_df = pd.concat([Y_train_df, Y_hat_df]).set_index('ds')

plot_df[['y', 'NBEATS', 'NHITS']].plot(ax=ax, linewidth=2)

ax.set_title('AirPassengers Forecast', fontsize=22)
ax.set_ylabel('Monthly Passengers', fontsize=20)
ax.set_xlabel('Timestamp [t]', fontsize=20)
ax.legend(prop={'size': 15})
ax.grid()

### Issue Severity

High: It blocks me from completing my task.

**Comment by Houyikai** [comment]
is this repo really work with python 3.10
